deepfence_server/model/registry.go

```diff
@@ -129,8 +138,20 @@ type Summary struct {
 	Tags            int `json:"tags"`
 }
 
+var queryRegistriesByType = `
+    MATCH (n:RegistryAccount{registry_type:$type})
+    WITH COUNT(distinct n) as registries
+	OPTIONAL MATCH (n:RegistryAccount{registry_type:$type})-[:HOSTS]->(i:ImageStub)-[:IS]-(m:ContainerImage)<-[:HOSTS]-(n)
+    WITH registries,
+        COUNT(distinct i.docker_image_name) AS repositories,
+        COUNT(distinct m.docker_image_id) AS images
+    OPTIONAL MATCH (s)-[:SCANNED]->()<-[:HOSTS]-(:RegistryAccount{registry_type:$type})
+    RETURN COLLECT(s.status) AS scan_status, repositories, images, registries
+	`
```

Comment:
Same as other 2 comments, we would benefit having a separate query to retrieve:
`WITH COUNT(distinct n) as registries` and then forget about it